### ISSUE TITLE
[Enhancement] Support VoiceOver in Calendar

### ISSUE THREAD

**Issue by MortenGregersen** [kirbydesign/designsystem#943]
*NOT Prioritized*
- [ ] For optimal support for VoiceOver this is needed.

*NOT UX Refined*
- [x] Doesn't need any UX.

*NOT Tech Refined*
- [ ] Decide how the calendar should be used with VoiceOver and if the name of the dates and months should be read, how do they translate?

**Is your enhancement request related to a problem? Please describe.**
When using VoiceOver (iOS) on a web app using Kirby right now, it's hard to navigate and you can't tell what date is selected.

**Describe the solution you'd like**
All dates in the calendar should have something a screen reader can read, that makes sense for blind people.

**Describe alternatives you've considered**
No alternatives considered.

**Additional context**
No additional context.

**Comment by MortenGregersen** [comment]
Add `aria-label` to every date with a text corresponding to Material Design calendar and/or iOS Calendar. The texts for navigating 'back' and 'forward' between month should be giving like in #942.
